decimal-macros/examples/test.rs

```diff
@@ -0,0 +1,17 @@
+#![feature(plugin)]
+#![plugin(decimal_macros)]
+extern crate decimal;
+
+fn main() {
+    let a = d128!(0.1);
```

Comment:
True, I should - was just kinda rushing to get it out the door and working :p I had so much trouble with it